src/views/assets/Cloud/Account/components/TimingPanel.vue

```diff
@@ -71,10 +71,9 @@ export default {
       form.validate((valid) => {
         if (valid) {
           btn.loading = true
+          this.$refs.form.$refs.form.dataForm.submitForm('form', false)
         }
       })
-      form.value.interval = parseInt(form.value.interval, 10)
-      this.$refs.form.$refs.form.dataForm.submitForm('form', false)
     },
     handleSubmitSuccess(res) {
       this.$emit('update:visible', false)
```

Comment:
The changes introduced here do not involve any irregularities or potential issues that could impact the functionality of this code snippet. There is no evident need for optimization at present but improvements can always be made with better practices and techniques.

Therefore, there were no noticeable adjustments to make from your provided source:

No modifications needed for accuracy in logic and function usage based on these checks, which are related strictly to syntax corrections rather than content updates or refactorings.

If you require further analysis or have specific queries regarding the structure and operation of the code, please let me know!